src/main/java/com/cleanroommc/groovyscript/compat/mods/tinkersconstruct/Casting.java

```diff
@@ -68,6 +71,7 @@ public boolean remove(ICastingRecipe recipe) {
             return true;
         }
 
+        @MethodDescription(description = "groovyscript.wiki.removeByOutput", example = @Example("item('minecraft:iron_ingot')"))
```

Comment:
`minecraft:iron_ingot` has already been removed, and so this will error. consider either marking this example as a comment, or changing it to  be eg `mincraft:gold_ingot`